src/utils/twoFactor.js

```diff
@@ -186,11 +187,22 @@ export class TwoFactor {
         const contractBytes = new Uint8Array(await (await fetch('/multisig.wasm')).arrayBuffer())
         const { accountId } = accountData
         const account = this.wallet.getAccount(accountId)
-        const accountKeys = await account.getAccessKeys();
+        // replace account keys & recovery keys with limited access keys; DO NOT replace seed phrase keys
+        const accountKeys = (await account.getAccessKeys()).map((ak) => ak.public_key)
+            .filter((k) => k.indexOf('9t9') === -1)
```

Comment:
@mattlockyer what does this check mean? What is 9t9?

Reply:
testing code, removed